clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10097,8 +10097,13 @@ void OffloadWrapper::ConstructJob(Compilation &C, const JobAction &JA,
     // clang-offload-wrapper
     //   -o=<outputfile>.bc
     //   -host=x86_64-pc-linux-gnu -kind=sycl
+#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
```

Comment:
I think you are just updating the comment here.
Would it be better to update it to: 
```
When -fpreview-breaking-changes is passed, it will be    
//   -format=spirv <inputfile1>.spv
//   -format=spirv <inputfile2>.spv and if not, then 
//   -format=spirv <inputfile1>.spv <manifest1>(optional)
//   -format=spirv <inputfile2>.spv <manifest2>(optional)
```

Adding include guards in comments seems uncommon to me.